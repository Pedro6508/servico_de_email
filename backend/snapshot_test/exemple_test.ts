import { assertSnapshot } from "https://deno.land/std@0.177.0/testing/snapshot.ts";

Deno.test("isSnapshotMatch", async function (t): Promise<void> {
  const first = {
    example: 123,
    hello: "world!",
  };

  const second = {
    example: 321,
    world: "hello!",
  };

  await assertSnapshot(t, first);
  await assertSnapshot(t, second);
});
