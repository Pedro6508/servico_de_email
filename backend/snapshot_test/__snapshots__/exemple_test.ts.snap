export const snapshot = {};

snapshot[`isSnapshotMatch 1`] = `
{
  example: 123,
  hello: "world!",
}
`;

snapshot[`isSnapshotMatch 2`] = `
{
  example: 321,
  world: "hello!",
}
`;
